README.md

```diff
@@ -49,11 +49,10 @@ app$layout_set(
   htmlDiv(id = "outputID")
 )
 
-app$callback(
-  function(x = input("inputID"), y = input("inputID", "type")) {
+app$callback(output=list("outputID"), inputs=list(list("inputID", "type")), user_function=
+  function(x, y) {
```

Comment:
This should just be a single input argument now, right?

Reply:
Each `id`/`property` pair is a single call to the `input` function.

Reply:
As shown above (or in my understanding of how we want it modified https://github.com/plotly/dashR/pull/51/files/5f89e8034c793e2ae37266b58251887542280667#r258077217) there's only one id/prop pair now, so this function should only take one arg, right?

Reply:
As we discussed, the `README.md` is now updated. I'll go ahead and trim out the declarations of `id` and `property` since they're implicit if unspecified.